### PR TITLE
PWN-3079 - hide devNet on builds for prod

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3079

## Description of Work

Make an option to hide or show DevNetbut hide it on ProdBuilds

### CODE REVIEW

**Review by 1sakov — APPROVED** [review]

File: app/src/main/res/layout/view_debug_drawer_configure_environment.xml [hunk 1]
```diff
@@ -39,4 +39,15 @@
         android:text="Enable auto update"
         tools:ignore="HardcodedText" />
 
+    <androidx.appcompat.widget.SwitchCompat
```
Comment (gslevinkov) [inline app/src/main/res/layout/view_debug_drawer_configure_environment.xml:42]: SwitchCompat -> Switch

File: app/src/main/res/layout/view_debug_drawer_configure_environment.xml [hunk 2]
```diff
@@ -39,4 +39,15 @@
...
+        android:id="@+id/enableProdEnvSwitch"
+        style="@style/WalletTheme.TextAppearance.Regular16"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:checked="true"
+        android:paddingHorizontal="16dp"
+        android:text="Enable DevNet env"
```
Comment (gslevinkov) [inline app/src/main/res/layout/view_debug_drawer_configure_environment.xml:50]: to strings?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/settings/ui/network/SettingsNetworkFragment.kt [hunk 3]
```diff
@@ -66,17 +67,20 @@ class SettingsNetworkFragment :
         presenter.setNewEnvironment(environment)
     }
 
-    override fun showEnvironment(environment: Environment) {
+    override fun showEnvironment(environment: Environment, isDevnetEnabled: Boolean) {
```
Comment (davran1001) [inline app/src/main/java/org/p2p/wallet/settings/ui/network/SettingsNetworkFragment.kt:70]: I think it's not good idea to have isDevNet booleans

Comment (davran1001) [reply app/src/main/java/org/p2p/wallet/settings/ui/network/SettingsNetworkFragment.kt:70]: maybe we will decide is it dev net or not in some class which take as parameter build flavor

**Review by davran1001 — APPROVED** [review]

**Pull request merged**